app/use_cases/subscribe_to_protocol.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class SubscribeToProtocol < ActiveInteraction::Base
+  string :protocol_name, default: nil
+  object :protocol, default: nil
+  object :person
+
+  # Watch out! IF you set a start date here (e.g. = Time.now.in_time_zone) it
+  # will set it once, and reuse THAT time everytime. I.e., it will not update
+  # the time when time passes.
+  time :start_date, default: nil
+
+  # Function to start a protocol subscription for a person
+  #
+  # Params:
+  # - protocol_name: the name of the protocol to start (optional)
+  # - protocol: the protocol to subscribe the person to (optional)
+  # - person: the person to start the protocol for
+  # - start_date: the date when the subscription should start
+  def execute
+    protocol = find_protocol
```

Comment:
Gaat dit goed als protocol ook een argument is die je meekrijgt? En ook als het wel goed gaat, misschien hernoemen om verwarring te voorkomen?